README.md

```diff
@@ -95,9 +98,21 @@ To use this mode, your SFCC instance must be configured to allow the origin of t
 }
 ```
 
+#### Allowing the extension in OCAPI settings
+
+To allow the extension to access product data in your SFCC environment, you will also need to add this to your SFCC OAuth client's `allowed_origins`:
+
+```json
+{
+  "allowed_origins": [
+    "https://product-selector.extensions.content.amplience.net"
+  ]
+}
+```
+
 ### SFCC (proxy)
 
-The extension works with 'list of text' properties and supports the following parameters:
+To use this mode, you will also need to ensure that "Allow same origin" is disabled within the [extension's settings](#Permissions). You will also need to build and host your own version of [sfcc-product-search-proxy](https://github.com/amplience/sfcc-product-search-proxy). The extension works with 'list of text' properties and supports the following parameters:
```

Comment:
Just to be nit picky. If you don’t use a registered extension and just ‘url’ you won’t get allow-same-origin so really it’s:

> To use this mode as a registered extension, you will also need to ensure that “Allow same origin” is disabled within the [extension’s settings](#Permissions). If you reference the extension using url this will happen automatically.

Reply:
Added this clarification.